### PR TITLE
Add usage with rails

### PR DESCRIPTION
**Pull request by riseshia**
`default_queue` is not work with Rails, and also there is no mention about barbeque adapter for rails. 😂

This pr has:

- Add notice about `default_queue` with Rails.
- Add explanation about active job adapter

Could you review this? @cookpad/dev-infra
cc @k0kubun

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,21 @@ message_id = "a3c653c1-335e-4d4d-a6f9-eb91c0253d02"
...
+Barbeque client has adapter for ActiveJob.
+
+```rb
+# config/environments/some_environment.rb
+Rails.application.config.active_job.queue_adapter = :barbeque
+```
+
+And everything will be ok. Don't forget to setup `config.application` and `config.endpoint` in somewhere.
+One more thing, `config.default_queue` option is meaningless with Rails.
+`default_queue` is the fallback option for enqueueing with not specified queue name.
```
Comment (eagletmt) [inline README.md]: "without queue name"

File: README.md [hunk 2]
```diff
@@ -40,6 +40,21 @@ message_id = "a3c653c1-335e-4d4d-a6f9-eb91c0253d02"
...
+However, ActiveJob always set default queue as 'default' internally,
+there is no place to work on. So Please use [`queue_as`](http://api.rubyonrails.org/classes/ActiveJob/QueueName/ClassMethods.html#method-i-queue_as) when you want to use different queue name.
```
Comment (eagletmt) [inline README.md]: "So **p**lease"

**Comment by riseshia** [comment]
Thanks!! fixed 54d623a

**Review by eagletmt — APPROVED** [review]

**Review by k0kubun — APPROVED** [review]
:+1:

**Comment by riseshia** [comment]
:bow:

**Pull request merged**